src/config/config.yml

```diff
@@ -15,6 +27,12 @@ invoke:
       getDefects:
         name: cvs-svc-defects
         mock: tests/resources/defects.json
+      getIvaDefectsByManual:
+        name: cvs-svc-iva-defects
+        mock: tests/resources/iva-defects.json
+      getIvaDefects:
+        name: cvs-svc-iva-defects
+        mock: tests/resources/iva-defects.json
```

Comment:
Just an observation, but as you can see iva-defects.json is being used here for the mock, rather than M1_flattened.json